Completely new version of E canteen app

CodeXdhruv opened the pull request:
I have made a completely new version of application which includes authentication with supabase and also tried to make a consistent ui of the application. Also changed the name of the application and named it "FoodieHub". I also faced an issue while running it on my android phone , the issue was that I was not able to login so please check it once in your android phone. Please review it and always open for suggestions.

CodeXdhruv committed:
Updation of project E canteen

CodeXdhruv committed:
feat: Transform E-Canteen-App into professional FoodieHub food ordering platform

Gautamsam3 commented:
thats amazing every functionality is working fine 

just you have given the option like quick demo login that was not working 

and instead of directly showing sign in/sign up page show them home page and at the time of checkout make them login

rest is great 10/10

CodeXdhruv commented:
Sir, I guess in commit "feat: Transform E-Canteen-App into professional FoodieHub food orderi.." which is in completely in new branch i have removed the quick demo  login button  and also i have added splash screen in the start. Please review it once.
